### PR TITLE
Revamp and polish docs

### PR DESCRIPTION
**Pull request by rjt-rockx**
- [x] Add a hero section to the landing page
- [x] Switch to grid layout instead of horizontal scroll
- [x] Use monochrome logo in navbar
- [x] Add disclaimer in footer
- [x] Use Overpass font
- [x] Polish scrollbar styling
- [x] Use neutral tone instead of slate
- [x] Ensure consistency when using opacity/text color to indicate secondary/tertiary text
- [ ] Ensure consistency with button styling
- [x] Add copy indicator to main button
- [ ] Add orange gradient/flair to the landing page
- [x] Re-add Discord link to nav
- [x] Use icons for nav
- [ ] Clarify what the npm install button does

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -34,7 +34,8 @@
 
 		%sveltekit.head%
 	</head>
+
 	<body>
-		%sveltekit.body%
+		<div style="display: contents">%sveltekit.body%</div>
```
Comment (TGlide) [inline src/app.html:39]: Why is this needed?

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -1,53 +1,50 @@
...
 	<div class="flex grow flex-col">
 		<slot />
 	</div>
+	<footer>
+		<div
+			class="flex flex-col items-start justify-between gap-2 px-6 py-4 lg:flex-row lg:items-center"
+		>
+			<span class="flex gap-1">
+				Inspired by <a
+					href="https://radix-ui.com/"
```
Comment (TGlide) [inline src/routes/+layout.svelte:43]: There's no way to know this is a link without hovering, maybe a dashed underline could work before hover?

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -15,77 +13,49 @@
...
-	}
-
-	function handleNext() {
-		const newIndex = Math.min(activeIndex + 1, sortedSchemas.length - 1);
-		wrapperEl.scrollTo({
-			left: newIndex * 600 + GAP * newIndex,
-			behavior: 'smooth',
-		});
+	function copyNpmCommand() {
+		navigator.clipboard.writeText(`npm install radix-ui-svelte`);
```
Comment (TGlide) [inline src/routes/+page.svelte:17]:
```suggestion
		navigator.clipboard.writeText(`npm install radix-svelte`);
```

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -15,77 +13,49 @@
...
+			<h1 class="text-4xl font-bold text-white lg:text-5xl">Don’t reinvent the wheel.</h1>
+			<p class="text-lg text-white opacity-75 lg:text-xl">
+				Unstyled, accessible components for building high‑quality design systems and web apps, now
+				in Svelte.
+			</p>
+			<div class="flex flex-row gap-4">
+				<button
+					on:click={copyNpmCommand}
+					class="text-md rounded bg-neutral-900 p-4 font-mono text-white transition hover:bg-neutral-800 active:translate-y-1"
+					>npm install radix-svelte</button
```
Comment (TGlide) [inline src/routes/+page.svelte:33]: I really like this! Would be cool to be able to alternate between npm, yarn and pnpm. Also we should probably have some feedback when clicking, such as a copy icon that changes to a check when clicked

File: src/routes/+layout.svelte [hunk 5]
```diff
@@ -1,53 +1,50 @@
...
 <main class="flex min-h-screen flex-col">
-	<nav class="flex items-center justify-between gap-2 border-b border-zinc-400 px-2 lg:px-4 py-2">
-		<a class="flex items-center gap-2 lg:text-xl font-sans font-bold" href="/">
+	<nav class="flex items-end justify-between px-6 py-4">
+		<a class="flex items-center gap-2 font-sans text-xl" href="/">
 			<img
-				class="w-7 h-7 lg:h-9 lg:w-9 rounded-sm object-contain"
-				src="/logo.svg"
+				class="h-8 w-8 rounded-sm object-contain"
+				src="/radix-svelte.svg"
```
Comment (TGlide) [inline src/routes/+layout.svelte:15]: I liked the orange logo here 🤔

File: src/routes/+page.svelte [hunk 6]
```diff
@@ -15,77 +13,49 @@
...
-				<ChevronLeft />
-			</button>
-			<button
-				class="button"
-				on:click={handleNext}
-				disabled={activeIndex === sortedSchemas.length - 1}
+		{#each sortedSchemas as [identifier, schema], idx}
+			{@const propsObj = getPropsObjForSchema(schema)}
+			<div
+				class={cn(
```
Comment (TGlide) [inline src/routes/+page.svelte:46]: Why use cn here?

File: src/routes/docs/[component]/+layout.svelte [hunk 7]
```diff
@@ -19,9 +19,9 @@
 	{@html theme}
 </svelte:head>
 
-<div class="flex grid-cols-12 flex-col gap-8 overflow-hidden py-6 lg:grid lg:px-4">
+<div class="flex grid-cols-12 flex-col gap-8 overflow-hidden py-6 lg:grid lg:px-6">
 	<div class="col-span-2">
-		<ul class="flex w-full overflow-x-auto px-4 pb-2 lg:block lg:px-0">
+		<ul class="flex w-full flex-col gap-2 overflow-x-auto p-2">
```
Comment (TGlide) [inline src/routes/docs/[component]/+layout.svelte:24]: There shouldn't be a gap here, IMO. MAkes it possible to click between links, resulting in a non-action, which is frustrating. I agree it can be visually strange, but I see it in other places too 
image

**Review by TGlide — COMMENTED** [review]
Thank you for the PR!!

Some notes:
- I really like the tagline, together with the install command and getting started button!
- Mobile has some small issues
- I personally prefer the old navbar for the mobile. That, or we should inclue a hamburger menu of some sorts. (actually I would love the current navbar but on the bottom, easier to access 🤔 IDK just thinking out loud)
- Grid is much more practical than what we have currently, but...
- This and some other cahnges made it look a bit of the colorful vibe it had. I'm not saying to undo it, but I say we need to find some way to add some flair to it.
- Overpass is interesting, it's growing on me. Maybe we should have a different mono font too?

File: src/routes/docs/[component]/+layout.svelte (hunk 7 above)
Comment (jca41) [reply src/routes/docs/[component]/+layout.svelte:24]: made similar change on this pr https://github.com/TGlide/radix-svelte/pull/109

i do agree it looks a bit weird on mobile for some reason.

Screenshot 2023-04-25 at 10 47 09

File: src/app.html (hunk 1 above)
Comment (rjt-rockx) [reply src/app.html:39]: > `%sveltekit.body%` — the markup for a rendered page. This should live inside a `<div>` or other element, rather than directly inside `<body>`, to prevent bugs caused by browser extensions injecting elements that are then destroyed by the hydration process. SvelteKit will warn you in development if this is not the case
https://kit.svelte.dev/docs/project-structure#project-files-src

File: src/routes/+layout.svelte (hunk 2 above)
Comment (rjt-rockx) [reply src/routes/+layout.svelte:43]: Yup, the goal is to have a consistent link style similar to the navigation links

File: src/app.html (hunk 1 above)
Comment (TGlide) [reply src/app.html:39]: Oh, TIL! Nice catch 😄

File: src/routes/+page.svelte (hunk 3 above)
Comment (rjt-rockx) [reply src/routes/+page.svelte:17]: I blame copilot lol

File: src/routes/+page.svelte (hunk 6 above)
Comment (rjt-rockx) [reply src/routes/+page.svelte:46]: Think I missed that, will remove

File: src/routes/docs/[component]/+layout.svelte (hunk 7 above)
Comment (rjt-rockx) [reply src/routes/docs/[component]/+layout.svelte:24]: Good point, will likely also move this to a menu as well. 

@jca41 I'll implement the changes here to avoid merge conflicts, thanks for pointing that out.

File: src/routes/+page.svelte (hunk 3 above)
Comment (TGlide) [reply src/routes/+page.svelte]: lol we've all been there

File: src/routes/+layout.svelte (hunk 5 above)
Comment (rjt-rockx) [reply src/routes/+layout.svelte:29]: I think the monochrome one looks sleeker tbh

File: src/routes/+page.svelte (hunk 4 above)
Comment (rjt-rockx) [reply src/routes/+page.svelte]: Copy icon done, will add additional package managers in a future PR

File: src/routes/docs/[component]/+layout.svelte (hunk 7 above)
Comment (rjt-rockx) [reply src/routes/docs/[component]/+layout.svelte]: I think a separate menu action for mobile devices would work better, will implement that in a separate PR

Comment (TGlide) [reply src/routes/docs/[component]/+layout.svelte]: @rjt-rockx Alright, seems good

File: package.json [hunk 8]
```diff
@@ -55,12 +55,15 @@
 	},
 	"dependencies": {
 		"@floating-ui/core": "^1.2.6",
-		"@floating-ui/dom": "^1.2.6"
+		"@floating-ui/dom": "^1.2.6",
+		"@fontsource/overpass": "^4.5.11",
+		"@fontsource/overpass-mono": "^4.5.11",
+		"@fontsource/source-code-pro": "^4.5.14",
+		"@iconify-json/lucide": "^1.1.82",
+		"@iconify-json/simple-icons": "^1.1.50"
```
Comment (TGlide) [inline package.json:63]: These should be devDependencies, since the `radix-svelte` package doesn't depend on them

File: src/routes/+page.svelte [hunk 9]
```diff
@@ -15,125 +16,68 @@
...
+					<ArrowRight class="inline-block h-5 w-5 text-white" />
+				</a>
+				<button
+					on:click={copyInstallCommand}
+					class="text-md group flex justify-between gap-4 rounded bg-zinc-900 p-4 font-mono text-white transition active:translate-y-0.5 active:bg-zinc-800 sm:shrink"
+					><span>npm install radix-svelte</span>
+					{#if copied}
+						<Check class="inline-block h-5 w-5 text-vermilion-500" />
+					{:else}
+						<Copy class="inline-block h-5 w-5 group-hover:text-vermilion-500" />
```
Comment (TGlide) [inline src/routes/+page.svelte:60]: Maybe the icon here could have a transition as well, since the button has it.

File: tailwind.config.ts [hunk 10]
```diff
@@ -40,5 +68,9 @@ export default {
 		},
 	},
 
-	plugins: [],
+	plugins: [
+		plugin(function ({ addVariant }) {
+			addVariant('hocus', ['&:hover', '&:focus']);
```
Comment (TGlide) [inline tailwind.config.ts:73]: lol I like it

**Review by TGlide — CHANGES_REQUESTED** [review]
Some other small changes:
- I think I preferred the paragraph after the title when it wrapped into two lines, makes it easier to read
- The logo text is a bit too spaced, maybe reducing to gap-1 could be better? Nitpicky, sorry
- When expanding some of the accordion items, the grid has a layout shift
- The Radix UI links should probably lead to new tabs, instead of replacing the current one
- When interacting with the components, the view docs underline glows. Is it on purpose?
- The read the docs button no longer has a hover state, I think it's better with one. 
- How do you feel about adding the orange gradient back on the component page docs only? Or something similar. If you feel it'll be bad, we can think of something to add flair later.

**Review by TGlide — APPROVED** [review]
Thank you, amazing work! LGTM 🎉
